ci-operator/config/kata-containers/kata-containers/kata-containers-kata-containers-main.yaml

```diff
@@ -0,0 +1,100 @@
+base_images:
+  base:
+    name: "4.6"
+    namespace: ocp
+    tag: base
+  qemu-builder:
+    name: ubuntu
+    namespace: ci
+    tag: "20.04"
+binary_build_commands: |
+  source ci/lib.sh
+  destdir="$(pwd)/_out/build_install"
+  mkdir -p "$destdir"
+  clone_tests_repo
+  pushd "${tests_repo_dir}"
+  # This builds Kata Containers and install on $destdir.
+  # Only QEMU is not built.
+  .ci/openshift-ci/build_install.sh "$destdir"
+
+  # This generates the Dockerfile used later in the pipeline to build QEMU.
+  .ci/openshift-ci/qemu-build-pre.sh
+  popd
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+images:
+- context_dir: tools/packaging
+  dockerfile_path: ./static-build/qemu/Dockerfile.ci
+  from: qemu-builder
+  inputs:
+    bin:
+      paths:
+      - destination_dir: ./static-build/qemu
+        source_path: /go/src/github.com/kata-containers/kata-containers/tools/packaging/static-build/qemu/Dockerfile.ci
+  to: kata-qemu
+- context_dir: .
+  dockerfile_path: ./Dockerfile.installer
+  from: base
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/kata-containers/kata-containers/_out
+      - destination_dir: .
+        source_path: /go/src/github.com/kata-containers/tests/.ci/openshift-ci/images/Dockerfile.installer
+      - destination_dir: .
+        source_path: /go/src/github.com/kata-containers/tests/.ci/openshift-ci/images/entrypoint.sh
+    kata-qemu:
+      paths:
+      - destination_dir: .
+        source_path: /tmp/qemu-static/kata-static-qemu.tar.gz
+  to: kata-installer
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.6"
+  namespace: ocp
+tests:
+- artifact_dir: /go/src/github.com/kata-containers/tests/_output/tests/openshift-e2e
+  as: e2e-tests
+  cron: 2 7 * * *
```

Comment:
Wainer explained elsewhere that the per-submit jobs is always there, but the github triggers have not been setup so they jobs don't run.

Reply:
@cpmeadors my ultimate goal is to have only the presubmit job to test PRs sent by the devs to kata-containers. However it may take a while until I figure out how to setup the trigger correctly, so I'd like to have the nightly job running in meanwhile.

Reply:
Ack